temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +293,160 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
+     * that require additional customization may use {@link #setSslContext(SslContext)} directly.
+     *
+     * @param keyCertChainInputStream - an input stream for an X.509 client certificate chain in PEM
+     *     format.
+     * @param keyPassword - the password of the key, or null if it's not password-protected.
+     * @param keyInputStream - an input stream for a PKCS#8 client private key in PEM format.
+     * @param fingerprints - a list of SHA1 fingerprints in hexadecimal form of the server's root
+     *     certificate.
+     * @throws SSLException - when it was unable to build the context.
+     */
+    public Builder setSslContextWith(
```

Comment:
This doesn't follow the standard builder pattern. The code should be:

setSslContext(SslContext.newBuilder()....build())

and all the variations of the methods should be done through the builder.

Reply:
I agree, we have way too many variations now, it's better to move it into a separate builder. The only reason I didn't do it initially is cause there is already SslContextBuilder in netty, so I didn't want to overload the name.

Reply:
Addressed.